ide/dlight.nativeexecution/external/binaries-list

```diff
@@ -15,3 +15,4 @@
 # specific language governing permissions and limitations
 # under the License.
 4B4DCA62F8C4A1954AE6D286955C36CC50B8CC3A exechlp-1.2.zip
+29797FF659132863DC32BCEB67817AB3F4CF4DD9 https://dist.apache.org/repos/dist/dev/netbeans/native/netbeans-nativeexecution/1-reb9e96d/nativeexecution-external-binaries-1-reb9e96d.zip nativeexecution-external-binaries-1-reb9e96d.zip
```

Comment:
If this becomes `https://archive.apache.org/...` when the actual release is done, this looks like a sane approach. Using `dist.apache.org` won't work to my understand, as `1-...` has to be removed once `2-....` is released, rendering old source code uncompileable.

Reply:
Thanks! Yes, that's the plan, for now at least.

At some point I hope we'll have proper release permalinks for this. Has been discussed elsewhere.